app/utils/report/templates/test.txt

```diff
@@ -60,6 +60,9 @@ Test Failures
       {%- for tc in group.test_cases %}
         {%- if 'FAIL' == tc.status %}
     * {{ tc.name }}:
+      {%- if tc.test_case_log_line %}
+        Test case HTML log: {{ storage_url }}/{{ group.file_server_resource }}/{{ group.lab_name }}/{{ group.boot_log_html }}#L{{ tc.test_case_log_line }}
```

Comment:
I believe the line number stored from the LAVA meta-data is not going to match what is in the HTML log, because some lines are removed (LAVA debug...).  For similar reasons, the line numbers will be different in the HTML and plain text log files.  So I think we need to be more specific and maybe call this field `html_log_line_number`, and translate the line numbers when creating the HTML log file.

Reply:
in the #216 when added the line number in the html file, I am doing in the way to add the number as close as possible to the lava callback log, if you didn't understand what I mean, see this link view-source:https://storage.staging.kernelci.org//kernelci/staging.kernelci.org/staging-20200421.0/arm/multi_v7_defconfig/gcc-8/lab-collabora/sleep-imx6q-sabrelite.html
you find for example line 8(L8) then 10(L10) but not line 9(L9).
Even if it's not the exact start line but in the exact logs, so I guess it's enough no ?
@gctucker @khilman

Reply:
Well I think it's a bug that some log line numbers are missing in the output HTML log to start with.  The numbers should be based in the HTML log, not where they came from.

Then the log line numbers stored in the test case should match exactly the line numbers in the HTML log.

All this data is known and handled by the backend code so it should only be a matter of fixing the logic.